opencti-platform/opencti-front/src/private/components/pir/pir_knowledge/PirKnowledge.tsx

```diff
@@ -34,12 +34,12 @@ interface PirKnowledgeProps {
 const PirKnowledge = ({ data }: PirKnowledgeProps) => {
   const pir = useFragment(knowledgeFragment, data);
   const pirId = pir.id;
-  const LOCAL_STORAGE_KEY = `PirSourcesFlaggedList-${pirId}`;
+  const LOCAL_STORAGE_KEY = `Pir-SourcesFlaggedList-${pirId}`;
```

Comment:
local storage key changed to clean the local storage and thus avoid having pir filters in the old format